### ISSUE TITLE
brew install git fails because missing curl.h

### ISSUE THREAD

**Issue by albertz** [Homebrew/linuxbrew#680]
https://gist.github.com/anonymous/9d9667f6d3f9c50c1283

Basically:

```
In file included from http.c:2:0:
http.h:6:23: fatal error: curl/curl.h: No such file or directory
compilation terminated.
```

So, I guess Curl is a missing dep.

**Comment by albertz** [comment]
After I did `brew install curl`, I now get another error: #682

**Comment by sjackman** [comment]
I'm on vacation until the new year. If this issue is still a problem for you after then, please bump this thread. Sorry for the delay.

**Comment by sjackman** [comment]
See issue should be fixed by ce169f5
